src/ApplicationParameters.php

```diff
@@ -9,12 +9,18 @@ final class ApplicationParameters
     private string $charset = 'UTF-8';
     private string $language = 'en';
     private string $name = 'My Project';
+    private string $email = 'support@example.com';
```

Comment:
I'd make it part of `ContactForm` service instead.

Reply:
done.